evadb/functions/function_bootstrap_queries.py

```diff
@@ -202,6 +202,18 @@
             MODEL 'yolov8n.pt';
         """
 
+stablediffusion_function_query = """CREATE FUNCTION IF NOT EXISTS StableDiffusion
+        IMPL '{}/functions/stable_diffusion.py';
+        """.format(
+    EvaDB_INSTALLATION_DIR
+)
+
+dalle_function_query = """CREATE FUNCTION IF NOT EXISTS StableDiffusion
```

Comment:
@Chitti-Ankith dalle_function_query isn't included in the list of queries that are loaded by default. I just added the query in case we need to use it at some point.

Reply:
@Chitti-Ankith Nevermind, you were talking about the function name. You're right.